GDAPI/GDAPI/Program.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace GDAPI
+﻿namespace GDAPI
```

Comment:
there's no reason for this change here. create a secondary pull request to clean up some of the code if you want.

Reply:
i just cleared unused dependencies, was that unnecessary ?